src/components/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+
+import React from "react";
+import { Document, Page, Text, View, StyleSheet } from "@react-pdf/renderer";
+
+export interface ParcelClients {
+    packing_datetime: string;
+    collection_centre: string;
+    collection_datetime: string;
+    voucher_number: string;
+    full_name?: string;
+    phone_number?: string;
+    address_1?: string;
+    address_2?: string;
+    address_town?: string;
+    address_county?: string;
+    address_postcode?: string;
+    delivery_instructions?: string;
+    index: number;
+    total: number;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "11px",
+    },
+    cardWrapper: { border: "1pt solid black", margin: "10px" },
+    bold: { fontFamily: "Helvetica-Bold" },
+    row: { display: "flex", flexDirection: "row" },
+    col: { flex: 1, margin: "5px" },
+});
+
+const ParcelCard: React.FC<{ datum: ParcelClients }> = ({ datum }) => {
+    return (
+        <View style={styles.cardWrapper}>
+            <View style={styles.row}>
+                <Text style={styles.col}>
+                    <Text style={styles.bold}>NAME:</Text> {datum.full_name}
```

Comment:
can these be `Name: ` with a style for `text-transform: uppercase` in the style? screen-readers do badly at all caps things iirc